dotCMS/src/main/webapp/html/portlet/ext/cmsmaintenance/index_stats.jsp

```diff
@@ -190,7 +184,11 @@ Map<String,ClusterIndexHealth> map = esapi.getClusterHealth();
 			         <td  align="center" class="showPointer"> <%= LanguageUtil.get(pageContext,"Closed") %> </td>
 			         <td  class="showPointer" ><%=idx%></td>
 			         <td><%=UtilMethods.webifyString(myDate) %></td>
-			         <td colspan="5">n/a</td>
+			         <td align="center">n/a</td>
+					<td align="center">n/a</td>
+					<td align="center">n/a</td>
+					<td align="center">n/a</td>
+					<td align="center">n/a</td>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid having style information in JSP files.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=3712629)